src/components/ConfigInsights/index.tsx

```diff
@@ -39,24 +38,54 @@ type Props = {
   configID: string;
 };
 
+const columns = [
+  {
+    key: "analysis",
+    label: "Name"
+  },
+  {
+    key: "age",
+    label: "Age"
+  }
+];
+
 function ConfigInsightsDetails({ configID }: Props) {
-  const [configInsights, setConfigInsights] = useState<ConfigTypeInsights[]>(
-    []
-  );
+  const [configInsights, setConfigInsights] = useState<
+    {
+      age: string;
+      analysis: React.ReactNode;
+    }[]
+  >([]);
   const { data: response = [], isLoading } =
     useGetConfigInsights<ConfigTypeInsights[]>(configID);
 
   useEffect(() => {
-    const data = response?.map((item) => {
-      return {
-        ...item,
-        sanitizedMessageHTML: sanitizeHTMLContent(item.message),
-        sanitizedMessageTxt: truncateText(
-          sanitizeHTMLContentToText(item.message)!,
-          500
-        )
-      };
-    });
+    const data = response
+      ?.map((item) => {
+        return {
+          ...item,
+          sanitizedMessageHTML: sanitizeHTMLContent(item.message),
+          sanitizedMessageTxt: truncateText(
+            sanitizeHTMLContentToText(item.message)!,
+            500
+          )
+        };
+      })
+      .map((item) => {
+        return {
+          age: relativeDateTime(item.first_observed),
+          analysis: (
+            <div
+              key={item.id}
+              data-html={true}
+              data-tip={item.sanitizedMessageTxt}
+              data-class="max-w-[20rem]"
```

Comment:
Why do we need this ?

Reply:
Config analysis string can be html with tags to render it properly in the tooltip we need these attributes.